src/main/java/org/gridsuite/caseimport/server/CaseImportController.java

```diff
@@ -36,16 +36,17 @@ public CaseImportController(CaseImportService caseImportService) {
         this.caseImportService = caseImportService;
     }
 
-    @PostMapping(value = "/cases", consumes = MediaType.MULTIPART_FORM_DATA_VALUE, produces = APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/cases/{caseName}", consumes = MediaType.MULTIPART_FORM_DATA_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
Not complient Rest !
caseName as `RequestParam` instead !
Param not required (optional) ?